packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -5542,10 +5542,14 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin, Res
                 includeSemantics: false,
                 child: UnmanagedRestorationScope(
                   bucket: bucket,
-                  child: Overlay(
-                    key: _overlayKey,
-                    clipBehavior: widget.clipBehavior,
-                    initialEntries: overlay == null ?  _allRouteOverlayEntries.toList(growable: false) : const <OverlayEntry>[],
+                  child: Semantics(
```

Comment:
Can you add a comment to explain why this is needed?

Reply:
This has been added, let me know if the comment works.  

There are also a lot of tests (100+) this breaks because this introduces a slight change to the semantics tree.  I don't think there's a way to automatically update the snapshot of the tree across all of them, so I'm working my way through updating these tests manually, which is taking quite a while.